docs/upgrade.html

```diff
@@ -74,6 +74,7 @@
         will return <code>0.0</code> in such cases to minimise the probability of breaking users who read the value of every client
         metric (via a <code>MetricsReporter</code> implementation or by calling the <code>metrics()</code> method).
         <code>org.apache.kafka.common.Metric#metricValue()</code> can be used to retrieve numeric and non-numeric metric values.</li>
+    <li>Every Kafka rate metric now has a corresponding cumulative count metric to simplify downstream processing.</li>
```

Comment:
Maybe worth mentioning the typical pattern? That is, `-count` becomes `-total`.

Reply:
@ijuma Thank you for the review, updated.